docs/content/en/contribute/software/prerequisite/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Software that is required
+description: Information about installing the tools before using Keptn Lifecycle Toolkit
+weight: 200
+---
+
+
+## Prerequisites
+
+* [**Docker**](https://docs.docker.com/get-docker/): a tool for containerization,
```

Comment:
Would comment somewhere that it could also be another container engine (e.g. Rancher Desktop)

Reply:
Added also I put the doc installation step link empty for now because it's not yet created. @thschue